src/modules/github/components/pull-list-item/pull-list-item.tsx

```diff
@@ -164,13 +209,33 @@ export const PullListItem = ({
           handlePullCheckChange(pull.number)
         }}
       />
-      <Link
-        href={pull.url}
-        target="_blank"
-        className="underline-offset-4 hover:underline"
+
+      <span
+        className="underline-offset-4 hover:underline cursor-pointer"
+        onClick={handleOpenDialog}
       >
-        <span>{pull.title}</span>
-      </Link>
+        {pull.title}
+      </span>
+
+      <Dialog
+        open={isDialogOpen}
+        onOpenChange={handleOpenChange}
+        title={pull.title}
+        children={<PullBody description={pull.body} />}
+        footer={
+          <>
+            <Button variant="outline" onClick={handleCancelClick}>
+              {translate('COMMON.ALERT_DIALOG_DEFAULT_CANCEL_BUTTON')}
+            </Button>
+            <Link href={pull.url} target="_blank">
+              <Button variant="primary">
+                {translate('COMMON.DIALOG_LINK_TO_PULL_REQUEST_BUTTON')}
+              </Button>
+            </Link>
+          </>
+        }
+      ></Dialog>
```

Comment:
You don't have to use `<Dialog>` twice in here if there are no children. You can just use it like `<Dialog />`

Reply:
ahh! okay!